aiodocker/utils.py

```diff
@@ -227,12 +217,12 @@ def clean_filters(filters: Mapping = None) -> str:
     return json.dumps(filters)
 
 
-def mktar_from_dockerfile(fileobject: BinaryIO) -> IO:
+def mktar_from_dockerfile(fileobject: Union[BytesIO, IO[bytes]]) -> IO[bytes]:
```

Comment:
I am not sure what to use here as `Tarfile.gettarinfo` expects fileobj as `BytesIO`

Reply:
Ok, let's keep `BytesIO` if there is no choice.

I can recommend making a PR for typeshed; they accept corrections easily and quickly if a patch is correct.